lib/goodreads_parser.py

```diff
@@ -289,9 +290,8 @@ def fullname_to_possible_aliases(self, fullname):
                     aliases.append("%s %s" % (title, surname))
             # Don't want the formats {ChristianName}, {Surname} and {ChristianName} {Lastname} in special cases
             # i.e. The Lord Ruler should never have "The Ruler", "Lord" or "Ruler" as aliases
-            if christian_name in self.COMMON_WORDS:
-                aliases.append('{0} {1}'.format(' '.join(parts), surname))
```

Comment:
You've removed this alias (here parts has had christian and surname removed, so is whatever is left) - which might be correct but right now I can't see when we would have wanted it!  with "The Lord Ruler"  christian_name would be "The", surname "Ruler", so we would have added an alias of "Lord Ruler" - which seems wrong.

So maybe you've just removed something that was never needed, just wanted to check.